accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -1906,13 +1911,20 @@ To type check a pattern `p` being matched against a value of type `M`:
         happen if a user-defined class has an override of `==` with a
         `covariant` parameter.*
 
+    The required type of `p` is `Object?`. (*The static checks mentioned above
```

Comment:
Typography nit: Put the parentheses inside the italics too, or just omit them since the italics already set the sentence aside.